New developer boards section and description for the HiHope SCDAY200 board (#15)
eclipse-oniro4openharmony/eclipse-oniro4openharmony.github.io: hh-scdayu200.md

---
title: HiHope HH-SCDAYU200
parent: Developer Boards
nav_order: 1
layout: default
---

# HiHope HH-SCDAYU200 Development Kit

## Introduction
Based on the Rockchip RK3568, this development kit integrates advanced features

> such as a dual-core architecture GPU and a high-performance NPU. Combined with
> the onboard quad-core 64-bit Cortex-A55 processor which has a frequency of up
> to 2.0GHz.

Reviewer: Lack of verb:
```suggestion
such as a dual-core architecture GPU and a high-performance NPU. Complemented by 
an onboard quad-core 64-bit Cortex-A55 processor which has a frequency of up to 2.0GHz, 
it ensures robust performance.
```

|-------------------|---------------|
| **HDMI**          | 1x HDMI2.0(Type-A), supports 4K at 60fps output |
| **MIPI**          | 2x MIPI interface, supports 1920x1080 at 60fps output |
| **eDP Interface** | 1x eDP interface, supports 2K at 60fps output |
| **I2S/TDM/PDM**   | 1x 8 channel I2S/TDM/PDM |
| **Ethernet**      | 2x GMAC(10/100/1000M) |
| **SDIO**          | Supports Wi-Fi 5G/2.5G, BT4.2 |
| **Camera Interface** | MIPI-CSI2, 1x4-lane/2x2-lane at 2.5Gbps/lane |
| **USB**           | 1x USB2.0 Host, Type-A; 1x USB3.0 Host, Type-A; 1x USB3.0 OTG |
| **M.2 Interface** | 4G LTE Module |
| **PCIe**          | 1x 2 Lanes PCIe3.0 Connector (RC Mode) |
| **SATA**          | 1x SATA3.0 Connector |
| **SDMMC**         | 1x Micro SD Card3.0 |
| **Buttons**       | 1x Vol+/Recovery; 1x Reset; 1x Power; 1x Vol-; 1x Mute |
| **RTC**           | 1x RTC |
| **Infrared**      | 1x IR |
| **LEDs**          | 3x LED |
| **Sensors**       | Bosch Sensortec BMA456, BMI270 and BMP581 |
| **Fan**           | 1x Fan |

## Building

To build Eclipse Oniro for this board the normal [quick build](quick-build.html)
procedure has to be used to fetch the needed source code and environment.

> At the actual build step, inside the Docker instance, the target device for the

Reviewer: ```suggestion
During the build step, inside the Docker instance, the target device for the
```

```bash
Bus 002 Device 009: ID 2207:5000 Fuzhou Rockchip Electronics Company "HDC Device"
...
Bus 001 Device 069: ID 0403:6001 Future Technology Devices International, Ltd FT232 Serial (UART) IC
```

Download the `flash.py` flashing tool from [Gitee](https://gitee.com/hihope_iot/docs/tree/master/HiHope_DAYU200/%E7%83%A7%E5%86%99%E5%B7%A5%E5%85%B7%E5%8F%8A%E6%8C%87%E5%8D%97/linux) using the following commands:

```bash
git clone https://gitee.com/hihope_iot/docs.git hihope_iot_docs
mkdir flash && cp -r hihope_iot_docs/HiHope_DAYU200/烧写工具及指南/linux/* flash/
chmod +x flash/flash.py flash/bin/flash.x86_64
```

To ensure proper device recognition, install the `udev` rule:

```bash
sudo cp flash/etc/udev/rules.d/85-rk3568.rules /etc/udev/rules.d/85-rk3568.rules
```
Then, either reload udev rules or reboot your system:

```bash
udevadm control --reload-rules
````

After this setup, running `flash/flash.py -q` should produce the following output, indicating readiness:

```bash
maskrom
```

To enable *programming mode* on the device, perform the following steps:

>  1. Press nad hold `VOL/RECOVERY` then `RESET` buttons.

Reviewer: ```suggestion
 1. Press and hold `VOL/RECOVERY` then `RESET` buttons.
```
